test/sqllogictest/alter.slt

```diff
@@ -73,7 +73,7 @@ ALTER MATERIALIZED VIEW mv SET CLUSTER default
 query error db error: ERROR: unknown cluster 'does_not_exist'
 ALTER MATERIALIZED VIEW mv SET CLUSTER does_not_exist
 
-query error db error: ERROR: ALTER SET CLUSTER are not supported
```

Comment:
So you are allowed to move a view to a cluster that does not exist?

Reply:
`other_cluster` exists, so the command is valid. The fact that `other_cluster` doesn't have any replicas is a different story, but that's fine as it's equivalent to reducing the replication factor of a cluster hosting materialized views to 0.